packages/twenty-front/src/modules/workflow/search-variables/components/VariableTagInput.tsx

```diff
@@ -130,6 +168,29 @@ export const VariableTagInput = ({
     onUpdate: ({ editor }) => {
       deboucedOnUpdate(editor);
     },
+    editorProps: {
+      handleKeyDown: (view, event) => {
+        if (event.key === 'Enter' && !event.shiftKey) {
+          event.preventDefault();
+
+          const { state } = view;
+          const { tr } = state;
+
+          // Insert hard break using the view's state and dispatch
+          const transaction = tr.replaceSelectionWith(
+            state.schema.nodes.hardBreak.create(),
+          );
+
+          view.dispatch(transaction);
+
+          return true;
+        }
+        return false;
+      },
+    },
```

Comment:
Why do we implement that logic? It seems that Tiptap should handle it, shouldn't it?

Reply:
Tip tap does not perform hardbreak on `Enter`. The behavior is not great by default

Reply:
Thanks for the info!